src/flick/suggestion/tests/test_private_suggestion.py

```diff
@@ -8,7 +8,8 @@
 
 
 class PrivateSuggestionTests(FlickTestCase):
-    SUGGESTION_URL = reverse("private-suggestion")
+    SUGGEST_URL = reverse("private-suggestion")
+    SUGGESTIONS_URL = reverse("private-suggestion-list")
```

Comment:
i'm curious what reverse does

Reply:
very good curiousity hehehe it basically gets the actual url we want!!! we define these views but the server will change based on if its localhost or public server, so we don't want to hard code something like http://localhost:8000/api/theurlwewant, so we give urls string names and then "reverse" the string to get the url!!!